src/server_manager/web_app/app.ts

```diff
@@ -755,6 +756,7 @@ export class App {
       view.monthlyOutboundTransferBytes =
           host.getMonthlyOutboundTransferLimit()?.terabytes * (10 ** 12);
       view.cloudLocation = host.getCloudLocation();
+      view.cloudId = host.getCloudId();
```

Comment:
Set the cloud name and icon instead, based on the Server type.

Reply:
I've moved the cloud ID to the account, which better maps it association.